### ISSUE TITLE
Remove Win32 directory from trunk

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#4587]
|[<img alt="khorn's avatar" src="https://avatars.githubusercontent.com/u/815352?s=50" width="50" height="50">](https://github.com/khorn)| @khorn reported|
|-|-|
|Trac ID|trac#4587|
|Type|task|
|Created|2010-07-14 20:52:24Z|
|Branch|https://github.com/twisted/twisted/tree/remove-win32-4587|

The /trunk/win32 directory in SVN was formerly used to create windows installers.  Since the completion of ticket [#11348](https://github.com/twisted/twisted/issues/11348), Twisted is no longer distributed using this install process, instead relying on distutils-compatible windows installer packages.

This directory should be removed, and any files in it that are still needed should be moved someplace else (twisted-icon.ico looks like it might be one).

<details><summary>Searchable metadata</summary>

```
trac-id__4587 4587
type__task task
reporter__khorn khorn
priority__normal normal
milestone__ 
branch__branches_remove_win32_4587 branches/remove-win32-4587
branch_author__exarkun exarkun
status__closed closed
resolution__fixed fixed
component__core core
keywords__ 
time__1279140744000000 1279140744000000
changetime__1301679000000000 1301679000000000
version__None None
owner__exarkun exarkun
cc__thijs
```
</details>

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_2"></a>|@thijstriemstra commented|
|-|-|

[#5010](https://github.com/twisted/twisted/issues/5010) was closed as a duplicate.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_3"></a>|@exarkun commented|
|-|-|

(In [31503]) Branching to 'remove-win32-4587'

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_4"></a>|@exarkun commented|
|-|-|

(In [31504]) Remove old win32 installer files

refs [#4587](https://github.com/twisted/twisted/issues/4587)

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_5"></a>|@exarkun commented|
|-|-|

Nothing fancy here...

**Comment by twisted-trac** [comment]
|[<img alt="mithrandi's avatar" src="https://avatars.githubusercontent.com/u/856430?s=50" width="50" height="50">](https://github.com/mithrandi)<a name="note_6"></a>|@mithrandi set owner to @exarkun|
|-|-|

Looks straightforward, please merge! :)

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_7"></a>|@exarkun set status to `closed`|
|-|-|

(In [31507]) Merge remove-win32-4587

Author: exarkun
Reviewer: mithrandi
Fixes: [#4587](https://github.com/twisted/twisted/issues/4587)

Remove the old Windows installer tools, long since unused.
